apps/balancer-tools/src/app/apr/pool/(components)/HistoricalCharts/HistoricalData/formatAPRChartData.tsx

```diff
@@ -75,12 +78,12 @@ export default function formatAPRChartData(
           line: { shape: "spline", color: "rgba(0,0,0,0);" } as const,
           type: "scatter" as PlotType,
         };
-      },
-    );
+      })
+    : [];
 
   const trimmedTokenTotalAprData = generateAndTrimAprCords(
-    apiResult.perRound,
-    (result) => result.apr.breakdown.tokens.total,
+    apiResult.perDay,
+    (result) => result[0].apr.breakdown.tokens.total,
```

Comment:
why get the first value here initially? 🤔

Reply:
perRound would return PoolStatData, perDay returns PoolStatData[]